src/overlays/actors/ovl_En_Torch2/z_en_torch2.c

```diff
@@ -579,18 +572,13 @@ void EnTorch2_Update(Actor* thisx, GlobalContext* globalCtx2) {
     // Updates Dark Link's "controller". The conditional seems to cause him to
     // stop targeting and hold shield if he's been holding it long enough.
 
-    phi_a2 = input->cur.button;
-    pad54 = input->cur.button ^ input->prev.button;
+    pad54 = input->prev.button ^ input->cur.button;
     input->press.button = input->cur.button & pad54;
-    phi_v0 = input->cur.button;
     if (input->cur.button & BTN_R) {
-        input->cur.button = phi_a2;
-        phi_a2 = ((sCounterState == 0) && (this->swordState == 0)) ? BTN_R : phi_v0 ^ BTN_R;
-        phi_v0 = phi_a2; // instruction mismatch
+        input->cur.button = ((sCounterState == 0) && (this->swordState == 0)) ? BTN_R : input->cur.button ^ BTN_R;
     }
     input->rel.button = input->prev.button & pad54;
-    input->prev.button = phi_v0 & 0x3FFF; // & ~(BTN_A | BTN_B)
-    input->cur.button = phi_a2;
+    input->prev.button = input->cur.button & 0x3FFF; // & ~(BTN_A | BTN_B)
```

Comment:
```suggestion
    input->prev.button = input->cur.button & ~(BTN_A | BTN_B);
```
this was probably waiting for matching to replace literals

Reply:
I tried putting it in and it didn't match lol

Reply:
Yup, a `(u16)` cast in front works